heat/core/tests/test_statistics.py

```diff
@@ -1157,14 +1157,15 @@ def test_percentile(self):
         q = 15.9
         for dim in range(x_ht.ndim):
             p_np = np.percentile(x_np, q, axis=dim)
+            p_np_keepdims = np.percentile(x_np, q, axis=dim, keepdims=True)
```

Comment:
The combination of sketched percentile with the new option to take percentile along a tuple of axes is not testes. Would that work or should we just throw an error?

Reply:
Great you noticed, I implemented the tests and found an error in `reshape` (fixed).